torch/distributed/tensor/_ops/utils.py

```diff
@@ -55,10 +50,9 @@ def wrapper(
 
 
 def register_op_strategy(
-    op, schema_info=None
-) -> Callable[[Callable[_P, _T]], Callable[_P, _T]]:
-    # pyre-fixme[2]: Parameter must be annotated.
```

Comment:
Actually, wait this is a decorator so that would erase typing info of the thing it wraps